dDatabase/CDClientDatabase/CDClientTables/CDMissionTasksTable.h

```diff
@@ -25,7 +27,7 @@ class CDMissionTasksTable : public CDTable<CDMissionTasksTable, std::vector<CDMi
 	// Queries the table with a custom "where" clause
 	std::vector<CDMissionTasks> Query(std::function<bool(CDMissionTasks)> predicate);
 
-	std::vector<CDMissionTasks*> GetByMissionID(uint32_t missionID);
+	std::vector<CDMissionTasks*> GetByMissionID(const uint32_t missionID);
```

Comment:
The corresponding function in the cpp file was left unchanged and no compiler error was detected.  This function can be removed and re-added if necessary.